### ISSUE TITLE
Add transcription of episodes for accessibility

### ISSUE THREAD

**Issue by mattstratton** [arresteddevops/ado-hugo#444]
We should add transcriptions of the episodes. We have done transcripts of episodes a few times a few years ago; there are various ways to generate them, but for this issue, I want to track our thoughts on how these transcripts will be on the episode pages.

This will have to be implemented in the code at [mattstratton/castanet](https://github.com/mattstratton/castanet), but I would like us to discuss how we would want to implement these.

Basically, it can be either that the transcripts are "linked" to in the show notes, or they are in the show notes directly

Some interesting things to read:

- https://www.temi.com/blog/podcast-transcription/
- https://podcast-accessibility.com/how-transcripts/
- https://github.com/mattstratton/castanet/issues/225

**Comment by mattstratton** [comment]
Oh, I did this. closing now.
